### PR TITLE
Fix/bundle import export missing product realtions

### PR DESCRIPTION
**Pull request by dvynograd**
Fixing missing product_catalog_relation during bundle import export

### CODE REVIEW

File: app/code/Magento/BundleImportExport/Model/Import/Product/Type/Bundle.php [hunk 1]
```diff
@@ -577,6 +578,41 @@ protected function populateInsertOptionValues($optionIds)
     }
 
     /**
+     * Insert realtions.
+     *
+     * @return \Magento\CatalogImportExport\Model\Import\Product\Type\AbstractType
+     */
+    protected function insertRelations()
```
Comment (maghamed) [inline app/code/Magento/BundleImportExport/Model/Import/Product/Type/Bundle.php:585]: this method should be made private from backward compatibility perspectives

**Review by maghamed — COMMENTED** [review]
could you please cover your changes with Unit test? 
We already have this test, which covers saveData method
https://github.com/magento/magento2/blob/develop/app/code/Magento/BundleImportExport/Test/Unit/Model/Import/Product/Type/BundleTest.php#L228

ideally it should be adopted to cover your changes

**Comment by ishakhsuvarov** [comment]
Hi @dvynograd 
Please check out the review comments if you wish to continue working on this.
Thanks

**Comment by ishakhsuvarov** [comment]
@dvynograd I am closing this PR for now because of inactivity.
Please check the code review comments and reopen if you wish to continue at any time.
Thank you for collaboration!

**Pull request closed without merging**